arduino/builder/testdata/TestLoadSketchFolderSymlinkSrc/.#sketch.ino

```diff
@@ -0,0 +1,2 @@
+void setup()
```

Comment:
what's this file for?

Reply:
Emacs recovery history file: see [here](https://github.com/arduino/Arduino/issues/4289) and [here](https://github.com/arduino/arduino-builder/issues/82)